Silicon/TigerlakePkg/Include/CpuRegs.h

```diff
@@ -100,6 +100,22 @@
 #define CPUID_FULL_FAMILY_MODEL_TIGERLAKE_ULT_ULX  0x000806C0
 #define CPUID_FULL_FAMILY_MODEL_TIGERLAKE_DT_HALO  0x000806D0
 
+#ifndef SLE_FLAG
+#ifndef STALL_ONE_MICRO_SECOND
+#define STALL_ONE_MICRO_SECOND 1
+#endif
+#ifndef STALL_ONE_MILLI_SECOND
+#define STALL_ONE_MILLI_SECOND 1000
+#endif
+#else // SLE FLAG
+#ifndef STALL_ONE_MICRO_SECOND
+#define STALL_ONE_MICRO_SECOND 0
+#endif
+#ifndef STALL_ONE_MILLI_SECOND
+#define STALL_ONE_MILLI_SECOND 0
+#endif
+#endif // SLE_FLAG

```

Comment:
It looks we only need add line 111~116 since SLE_FLAG is not defined and used for TGL.

Reply:
If necessary, we can define these only
```
#define STALL_ONE_MICRO_SECOND 1
#define STALL_ONE_MILLI_SECOND 1000
```

Reply:
I should keep 
#define STALL_ONE_MICRO_SECOND 1
#define STALL_ONE_MILLI_SECOND 1000
or
#define STALL_ONE_MICRO_SECOND 0
#define STALL_ONE_MILLI_SECOND 0
?

Reply:
it should be the first one.

Reply:
Done, thanks @myguitar and @gdong1 for the review.